media/css/firefox/releasenotes.less

```diff
@@ -447,6 +447,8 @@ article {
     .section-items {
         > li {
             padding-left: 0;
+            word-wrap:break-word;
+            width:50%;
```

Comment:
I'd take it further and suggest removing the left padding and setting the tags to <code>display:block</code>, <code>position:static</code>, and <code>width:auto</code> so they just stack up. Seems like a simpler fix and will be much more readable on small screens without squeezing the text into columns.

![releasenotes](https://cloud.githubusercontent.com/assets/205591/9041738/9dfa8c38-39be-11e5-88e2-cacc88dcffee.jpg)